### PR TITLE
new view with auth and form

### PR DESCRIPTION
**Pull request by sashashakun**
Pull request with auth form and some view improvements

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -3,3 +3,82 @@
  *= require jquery/dist/jquery.js
  *= require bootstrap/dist/js/bootstrap.js
  */
+
+ var lock = new Auth0Lock('pZJp3rVJ8wRvVLrX30HEOOHZ3HSEASec', 'hexlet-interviews.eu.auth0.com');
```
Comment (Shamash2014) [inline public/js/index.js:7]: Maybe better create credentials.js and import it here and use secret and callback properties? Also credentials.js could obtain this params from ENV, so no one will get them)

Comment (sashashakun) [inline public/js/index.js:7]: You are absolutely right)

Comment (Nitive) [inline public/js/index.js:7]: Do not you think it have to be fixed ASAP?

Comment (sashashakun) [inline public/js/index.js:7]: I think we will remove auth with third-party service before next interview, and now our app turned off

**Pull request merged**